### PR TITLE
Improve offline and online commander communication - Closes #6082

### LINKED ISSUE #6082: Improve offline and online commander communication
### Description

The commander bootstrap provide online and offline communication, which requires a base IPC command. This base IPC command needs to be refactored to work without requiring to use the application initialisation.

### Acceptance Criteria

- Refactor base IPC command in commander bootstrap command
- Remove unwanted base IPC command extension from bootstrapping template

### Additional Information
- Once again revisit the offline and online commander feature

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #6082 

### How was it solved?

- Refactored base ipc
- Refactored bootstrapped commands

### How was it tested?

1. `cd lisk-sdk`; `yarn; yarn build`
2. `cd commander; yarn link`
3. `mkdir /tmp/test_lisk; cd /tmp/test_lisk`
4. `../lisk-sdk/commander/bin/run init`
5. `yarn link lisk-commander`
6. `./bin/run`
7. `npm run test`

### CODE REVIEW

File: commander/src/bootstrapping/commands/transaction/sign.ts [hunk 1]
```diff
@@ -12,19 +12,183 @@
...
+		flags,
+		registeredSchema,
+		transactionHexStr,
+		flags['network-identifier'],
+		keys,
+	);
+	return signedTransaction;
+};
+
+const signTransactionOnline = async (
```
Comment (shuse2) [inline commander/src/bootstrapping/commands/transaction/sign.ts:132]: for online, is there any specific reason why `client.sign` cannot be used?

File: commander/src/utils/transaction.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	const { id, asset, ...txWithoutAsset } = transaction;
+	const txJSON = codec.toJSON(schema.transaction, txWithoutAsset);
+	return {
+		...txJSON,
+		asset: assetJSON,
+		id: Buffer.isBuffer(id) ? id.toString('hex') : undefined,
+	};
+};
+
+export const getApiClient = async (
```
Comment (shuse2) [inline commander/src/utils/transaction.ts:101]:
```suggestion
export const getAPIClient = async (
```

File: commander/src/bootstrapping/commands/transaction/create.ts [hunk 3]
```diff
@@ -44,7 +76,146 @@ const isSequenceObject = (
...
+	return validateAndSignTransaction(
+		transaction,
+		registeredSchema,
+		flags['network-identifier'],
+		passphrase,
+		flags['no-signature'],
+	);
+};
+
+const createTransactionOnline = async (
```
Comment (shuse2) [inline commander/src/bootstrapping/commands/transaction/create.ts:176]: why not use `client.create` function for online?

**Review by shuse2 — CHANGES_REQUESTED** [review]
overall looks good to me, some questions regards to the usage of api client

**Review by shuse2 — APPROVED** [review]

**Review by nazarhussain — APPROVED** [review]

**Pull request merged**